src/cgame/cg_players.c

```diff
@@ -2999,6 +3005,85 @@ void CG_Player(centity_t *cent)
 	// set blinking flag
 	CG_AddRefEntityWithPowerups(&head, cent->currentState.powerups, ci->team, &cent->currentState, cent->fireRiseDir);
 
+	// show blooded face
+	{
+		int damagedState = -1;
+		// show blooded face for teammates depending on their health
+		if (ci->team == cg.predictedPlayerState.teamNum)
+		{
+			if (cent->currentState.eFlags & EF_DEAD)
+			{
+				damagedState = 2;
+			}
+			else if (ci->health > 0)
+			{
+				if (healthPct <= 0.30)
+				{
+					damagedState = 2;
+				}
+				else if (healthPct <= 0.50)
+				{
+					damagedState = 1;
+				}
+				else if (healthPct <= 0.80)
+				{
+					damagedState = 0;
+				}
+			}
+		}
```

Comment:
It is inconcistent with the value set for client https://github.com/etlegacy/etlegacy/blob/17ea8b747152254852b09127ac61006bdae14f2e/src/cgame/cg_draw_hud.c#L604-L619

Reply:
> It is inconcistent with the value set for client

Good eye! That's on purpose tho after trying them out a lot, I think that my values are better - they are only visible for others as well, so the difference won't be noted.

I also used `cgs.media.hudDamagedStates[3]` for a while, but have found that it looks off and too intrusive - especially because the neck and the rest of the body won't ever be blooded, but only the face.

I think they blend in much better without it.

Let me know if this suffices for you, or if you want to still change it.